heroku-deploy/src/main/java/com/heroku/sdk/deploy/lib/deploymemt/Deployer.java

```diff
@@ -22,24 +20,28 @@
 public final class Deployer {
 
     public static boolean deploy(String apiKey, String clientName, String clientVersion, DeploymentDescriptor deploymentDescriptor, OutputAdapter outputAdapter) throws IOException, InterruptedException {
-        HerokuAPI herokuApi = new HerokuAPI(apiKey);
         HerokuDeployApi herokuDeployApi = new HerokuDeployApi(clientName, clientVersion, apiKey);
 
-        Source source = herokuApi.createSource();
-        Source.Blob sourceBlob = source.getSource_blob();
+        SourceBlob sourceBlob;
+        try {
+            sourceBlob = herokuDeployApi.createSourceBlob();
+        } catch (HerokuDeployApiException e) {
+            outputAdapter.logError("Could create source blob: " + e.getMessage());
```

Comment:
```suggestion
            outputAdapter.logError("Couldn't create source blob: " + e.getMessage());
```

Reply:
Fixed in [5e2b396](https://github.com/heroku/heroku-maven-plugin/pull/224/commits/5e2b396055b59de9fe6668ed9b482f56559de4e6)

Reply:
(I wouldn't normally use contractions either, but the Simplified Technical English recommendations and the Salesforce/Heroku style guides say to fwiw)